### PR TITLE
fix: Merge the same translation words #108

### PR DESCRIPTION
**Pull request by turbodog03**
### added(fixed) translations：
image
image
image

### code changes:
* I found some blocks like "token:Application", but there're no "token" section in en.json or zh.json files. so I edited them to be like "general:Application", and add their translation into the two .json files.
* I merged all statements in the two .json files that include "code" into one section named "code".

### CODE REVIEW

**Comment by hsluoyz** [comment]
@Kininaru plz review.

**Review by Kininaru — COMMENTED** [review]
Please do not add `package-lock.json`. Other changes look fine.

**Comment by hsluoyz** [comment]
@turbodog03 remove Merge commits:

image

**Review by Kininaru — COMMENTED** [review]
Please remove `web/package-lock.json`. Other changes look fine.

File: web/.gitignore [hunk 1]
```diff
@@ -21,3 +21,5 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+package-lock.json
```
Comment (hsluoyz) [inline web/.gitignore:25]: Don't change this

File: .gitignore [hunk 2]
```diff
@@ -24,3 +24,4 @@ tmpFiles/
 logs/
 lastupdate.tmp
 commentsRouter*.go
+
```
Comment (hsluoyz) [inline .gitignore:27]: Don't change this

File: web/src/auth/SignupPage.js [hunk 3]
```diff
@@ -131,17 +131,15 @@ class SignupPage extends React.Component {
       return null;
     }
 
-    const required = signupItem.required;
```
Comment (hsluoyz) [inline web/src/auth/SignupPage.js:134]: Don't change these

**Review by hsluoyz — CHANGES_REQUESTED** [review]
Don't change anything if you don't know how it works

**Pull request closed without merging**